src/interpreter/main.cpp

```diff
@@ -6,7 +6,7 @@
 using namespace Opcode;
 
 int main() {
-    unsigned char bytecodes[] = {BCONST_F, DUP, BR_F, 0x02, 0x00, 0x00, 0x00, POP, BCONST_T, PRINT, HALT};
+    unsigned char bytecodes[] = {HALT};
```

Comment:
maybe add a note that this is temporary (I presume?)

Reply:
Agreed.